### DISCUSSION TITLE
How to get full article in rss feed in current theme?

### DISCUSSION THREAD

**Discussion by Natenom** [razonyang/hugo-theme-bootstrap#601] [Q&A] [answered]
With the commid https://github.com/razonyang/hugo-theme-bootstrap/commit/01998870b9fcb56e48ed0188b3685ce695fff7a4 the index.xml contains excerpts only. Before that commit I could get full blog post in index.xml with using `.Content | html` in blog/layouts/_default/rss.xml

Now that doesn't work anymore. From how I understand the logic of the new rss.html, which calls _internal/_default/rss.xml it should work.

Can you please help?

**Comment by razonyang** [comment, marked as answer]
Sorry for the inconvenient. I think it's because of the Hugo layouts look up order. The `layouts/index.xml` has priority over `layouts/_default/rss.xml`.

I just push a patch for this 8445b4becbfd7ea9c51c1ddb366351d118321281, should be fixed.
